how-to-use-azureml/automated-machine-learning/subsampling/auto-ml-subsampling-local.yml

```diff
@@ -0,0 +1,8 @@
+name: auto-ml-subsampling-local
```

Comment:
This file is not in AzureMlCli and should not be pushed to GitHub.
It is generated as part of running the associated notebook.